### PR TITLE
[ERAD] Add test cleansing error is raise when input has nans

### PR DESCRIPTION
**Pull request by aperezhortal**
This PR adds a test for the cleansing function that checks that an error is raised when input has nans.

### CODE REVIEW

File: pysteps/tests/test_utils_cleansing.py [hunk 1]
```diff
@@ -174,3 +174,12 @@ def test_detect_outlier_multivariate_local():
     V[-1] = (-3, 3)
     outliers = cleansing.detect_outliers(V, 4, coord=X, k=50)
     assert outliers.sum() == 2
+
+def test_value_error_is_raise_when_input_has_nan():
+    cor = np.ones((3, 1))
+    iarr = np.ones((3, 1))
```
Comment (aperezhortal) [inline pysteps/tests/test_utils_cleansing.py]: What about using more descriptive variable names? 

`input_array`

File: pysteps/tests/test_utils_cleansing.py [hunk 2]
```diff
@@ -174,3 +174,12 @@ def test_detect_outlier_multivariate_local():
...
+    scale = 20
+
+    iarr[1,0] = np.nan
+    with pytest.raises(ValueError):
+        cleansing.decluster(cor, iarr, scale)
```
Comment (aperezhortal) [inline pysteps/tests/test_utils_cleansing.py]: Please, add a new line at the end.

**Review by aperezhortal — COMMENTED** [review]
@dnerini , what do you think? Is it ready?

**Review by dnerini — CHANGES_REQUESTED** [review]
The black test is failing, otherwise this is good to go!

**Review by dnerini — APPROVED** [review]

**Pull request merged**